sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StatefulProcessorHandleImpl.scala

```diff
@@ -112,6 +116,10 @@ class StatefulProcessorHandleImpl(
 
   private var currState: StatefulProcessorHandleState = CREATED
 
+  private def updateMetric(metricName: String): Unit = {
```

Comment:
[nit] Maybe `incrementMetric` is a better name as we always add 1.

Reply:
Done